cli/internal/cmd/init_test.go

```diff
@@ -156,30 +149,27 @@ func TestInitialize(t *testing.T) {
 			masterSecretShouldExist: true,
 			wantErr:                 true,
 		},
-		"empty id file": {
+		"empty state file": {
 			provider:      cloudprovider.GCP,
-			idFile:        &clusterid.File{},
 			stateFile:     &state.State{},
```

Comment:
nit: use state.New()

Reply:
IMO you wouldn't want to use constructors in the tests here, as they could potentially hide which values the struct is being populated with. In this case, it's not a problem, as right now it only fills in the version. But I think we should generally avoid using constructors in tests

Reply:
mmh but then add the version. The test might in theory error only because the version is missing.